executor/executor_ddl.go

```diff
@@ -72,6 +72,11 @@ func (e *DDLExec) Next() (*Row, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	// Update InfoSchema in TxnCtx, so it will pass schema check.
```

Comment:
add a TODO ?
schema check should not check schema change, and schema renew should not be automatically when schema check fail.